src/Plugin.php

```diff
@@ -55,6 +73,9 @@ public function __construct(array $config = [])
             throw new \InvalidArgumentException("Missing required configuration key 'nicks'");
         }
         $this->setNicks($config['nicks']);
+        if (!empty($config['recovery'])) {
```

Comment:
Ah, missed that, you're right. Never mind, ignore me. This is why I shouldn't review code before I've finished my first coffee of the morning.